app/views/posts/index.html.erb

```diff
@@ -1,11 +1,50 @@
-<h1>Posts</h1>
+<%= stylesheet_link_tag 'posts_index' %>
 
+<div class="user-show-container">
+  <div class="user-image-container">
+    <img src="<%= @user.photo %>" alt="<%= @user.name %>">
+  </div>
+
+  <div class="user-details-container">
+    <h1><%= @user.name %></h1>
+    <p class="post-count">Number of posts: <%= @user.posts.count %></p>
+  </div>
+</div>
+
+
+<div class="user-posts-container">
 <ul>
-  <% @posts.each do |post| %>
-    <li>
-      <%= link_to post.title, user_post_path(@user, post) %>
-    </li>
+
+<% @user.posts.each do |post| %>
+  <li>
+  <h2>
+  <% post_count = @user.posts.count %>
+  <% if post_count.zero? %>
+    No Posts
+  <% else %>
+    <% @user.posts.each_with_index do |post, index| %>
+      <%= link_to "Post ##{index + 1}", user_post_path(@user, post) %><% if index < post_count - 1 %>, <% end %>
```

Comment:
- [ ] This logic is displaying all the titles of all posts created by a user as you'll see below but on the wireframe this is not the case. We are advise to adhere to the layout of the wireframe for this specific.
![Screenshot from 2023-11-16 22-47-07](https://github.com/Xghayor/threads-rails/assets/63087491/d171dd1e-89d2-440b-8395-e1fa5857b8fd)